src/pages/index.tsx

```diff
@@ -1,59 +1,70 @@
-import { Button } from '@mui/material';
-import { useRouter } from 'next/router';
+import { Avatar, Button } from '@mui/material';
+import Link from 'next/link';
 import { useTranslation } from 'next-i18next';
-import { Suspense, useEffect } from 'react';
 import { useAuthState } from 'react-firebase-hooks/auth';
 
-import { auth, logout } from '@/app/components/FireBase';
-import { CustomSchema } from '@/entities/CustomSchema';
-import { fetchSchemaByAPI } from '@/features/SideBar/api/shemaByAnyAPI';
-import { DocumentSchemaLazy } from '@/features/SideBar/ui/DocumentSchema.lazy';
-import homeStyles from '@/pages/home.module.css';
-import { getCoreServerSideProps, SSRPageProps } from '@/shared/lib/ssr';
+import { auth } from '@/app/components/FireBase';
+import { DescDeveloper1 } from '@/features/SideBar';
+import { DescDeveloper2 } from '@/features/SideBar/ui/DescDeveloper2';
+import { DescDeveloper3 } from '@/features/SideBar/ui/DescDeveloper3';
```

Comment:
You can use one componet with 2 property alt and src, and save this property as array, for example:
[{dev1}, {dev2}, {dev3}] and then use map for render this components